docs/resources/callback_resource.md

```diff
@@ -66,6 +66,7 @@ resource "clumio_callback_resource" "example" {
 - **id** (String) The ID of this resource.
 - **properties** (Map of String) Properties to be passed in the SNS event.
 - **protect_config_version** (String) Clumio Protect Config version.
+- **protect_dynamodb_version** (String) Clumio DynamoDB Protect version.
 - **protect_ebs_version** (String) Clumio EBS Protect version.
 - **protect_ec2_mssql_version** (String) Clumio EC2 MSSQL Protect version.
 - **protect_enabled** (Boolean) Is Clumio Protect enabled.
```

Comment:
Should we denote somewhere that without providing these values, the latest versions will be used?

Reply:
This resource will not be used by the users directly and this resource will be part of the module which will have the values populated. No. The latest versions will not be used. These will have to be provided.

Reply:
Got it, it was always this way then right? In other words, even previously the callback resource needed to have the versions provided?

Reply:
Yes.